riscv/csrs.cc

```diff
@@ -1428,8 +1428,6 @@ vector_csr_t::vector_csr_t(processor_t* const proc, const reg_t addr, const reg_
 
 void vector_csr_t::verify_permissions(insn_t insn, bool write) const {
   require_vector_vs;
-  if (!proc->extension_enabled('V'))
```

Comment:
I would've expected this to be replaced with `!has_any_vector()`, but I'm guessing you left it out because some other mechanism has the same effect?

Reply:
I think `require_vector_vs` would do it, since `status_csr_t` masks the vs bits with `has_any_vector()` now. It's a roundabout check

Reply:
I was thinking that VS might be implemented even when `!has_any_vector()`, which is permitted by the ISA.  I forgot we don't do that in Spike.  So this is fine.